pkg/config/config.go

```diff
@@ -1874,6 +1875,16 @@ func (c *CustomAttributeMap) Decode(value string) error {
 	return nil
 }
 
+func (c *CustomAttributeMap) ToDataMap() (d data.Map) {
+	d = data.Map{}
+
+	for k, v := range *c {
+		d[k] = v.(string)
```

Comment:
If `v` is not a string this could `panic`. Perhaps now it's fine, but in the future might not. I'd suggest to check the assertion:
```suggestion
		valStr, isStr := v.(string)
		if !isStr{
			continue
		}

		d[k] = valStr
```